src/Integration/Aws/DynamoDbSession/.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 10
+      matrix:
+        php: ['7.2', '7.3', '7.4']
+
+    steps:
+      - name: Set up PHP
+        uses: shivammathur/setup-php@2.1.2
+        with:
+          php-version: ${{ matrix.php }}
+          coverage: xdebug
+          ini-values: xdebug.overload_var_dump=1
+          tools: prestissimo
```

Comment:
These are all old values. Look at dev-master for new values in this file.

Reply:
updated